NEWS.md

```diff
@@ -1,6 +1,8 @@
-
 # devel
 
+* `pkg_source()` now checks if biocViews is empty or not before assigning a 
```

Comment:
`pkg_source()` is not an exported function, so the user does not need to know about it. This could be something like:
```
* `package_info()` and `session_info()` now detect locally installed packages correctly
   if they have an empty `biocViews` field in `DESCRIPTION (#25, @llrs)
```

Reply:
Adopted this, but should I follow the (number, contributor) or the previously used (contributer, number)?

Reply:
Right, according to http://style.tidyverse.org/news.html#before-release in is (person, number(s)).